lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_xla_type.cpp

```diff
@@ -114,7 +116,9 @@ at::Tensor addmm(const at::Tensor& self, const at::Tensor& mat1,
       !at::native::is_floating_point(self) ||
       !at::native::is_floating_point(mat1) ||
       !at::native::is_floating_point(mat2)) {
-    return AtenXlaTypeDefault::addmm(self, mat1, mat2, beta, alpha);
+    return at::native::call_fallback_fn<&ltc_eager_fallback,
```

Comment:
can you add comments on the particular reason for each of these fallbacks?  (e.g. if it is globally a problem lowering them to TS, or if there is an nvfuser-specific issue, or something else)?

Reply:
I added a comment for max pooling operations, since the reason is solid for those. For the rest, it's quite simple - we've never seen those versions in any real model and I think it just makes sense to inherit that knowledge here instead of making nvfuser (or other backends) to implement matrix multiplication on integers, for example.